lib/Importer/ONNXModelLoader.cpp

```diff
@@ -256,6 +256,96 @@ bool ONNXModelLoader::loadOperator(const ONNX_NAMESPACE::NodeProto &op) {
     return true;
   }
 
+  if (typeName == "Slice") {
+    auto data = getNodeValueOrCreateConstantByName(op.input(0));
+    auto dims = data.dims();
+    auto numDims = dims.size();
+
+    // Attributes 'starts' and 'ends' are mandatory and must be consistent.
+    assert(dict.count("starts") && "Slice: attribute 'starts' is mandatory.");
+    assert(dict.count("ends") && "Slice: attribute 'ends' is mandatory.");
+    auto starts = getShape<ssize_t>(dict["starts"]);
+    auto ends = getShape<ssize_t>(dict["ends"]);
+    assert((starts.size() == ends.size()) &&
+           "Slice: 'starts' and 'end' arrays must have the same size.");
```

Comment:
Do `starts` and `ends` have to be the same size as `numDims`?  I can't tell from the spec.

Reply:
Well, the spec is very unclear and needs many clarifications. This is my interpretation that when there is a 'starts' integer, there *must* be a 'ends 'integer. Otherwise, for instance, if one 'ends' integer is missing, we don't know which axis has been omitted for 'ends' (it can be any).

Reply:
I think `starts` and `ends` must have the same size. But their size can be smaller or greater than `numDims` (it depends on `axes`).